src/skills-builder/skills-builder-modal/view-results/ViewResults.jsx

```diff
@@ -138,25 +140,39 @@ const ViewResults = () => {
       </Row>
     ) : (
       <Stack gap={4.5} className="pb-4.5">
-        <Alert
-          variant="success"
-          icon={CheckCircle}
-        >
-          <Alert.Heading>
-            {formatMessage(messages.matchesFoundSuccessAlert)}
-          </Alert.Heading>
-        </Alert>
-
-        <RelatedSkillsSelectableBoxSet
-          jobSkillsList={jobSkillsList}
-          selectedJobTitle={selectedJobTitle}
-          onChange={handleJobTitleChange}
-        />
+        { showMatchesFoundAlert && (
+          <Alert
+            variant="success"
+            icon={CheckCircle}
+          >
+            <Alert.Heading>
+              {formatMessage(messages.matchesFoundSuccessAlert)}
+            </Alert.Heading>
+          </Alert>
+        )}
+        { /* This should just pass the useInteractiveBoxSet flag to the component */ }
+        { useInteractiveBoxSet ? (
```

Comment:
Same as the `useProgressive` boolean. This would probably be better off written as `isInteractiveBoxSet` or `showInteractiveBoxSet` to avoid devs thinking it is a hook

Reply:
changed to isInteractiveBoxSet.